tests/db_access/prepared_statements.cpp

```diff
@@ -649,9 +649,20 @@ go_bandit([](){
         });
 
         it("can work with prepared statement helper functions - psReadQuery (invalid data types)", [&](){
-            AssertThrows(PreparedStatementTypeError, psReadQuery("SELECT `id`, `blob`, `binary`, `varbinary` FROM `test_superior_sqlpp`.`binary_data` ORDER BY `id` LIMIT 1", 
+            AssertThrows(PreparedStatementTypeError, psReadQuery("SELECT `id`, `blob`, `binary`, `varbinary` FROM `test_superior_sqlpp`.`binary_data` ORDER BY `id` LIMIT 1",
                 connection, [&](int , int, const BlobData &, const BlobData &) {}));
         });
+
+        it("can work with psQuery helper function", [&](){
+            const int target = 5;
+            int counter = 0;
+            psQuery("SELECT ?", connection,
```

Comment:
This being very useful function, please consider adding another, more real-world use.